src/utils/urlPaths.ts

```diff
@@ -12,6 +12,7 @@ export enum UrlPathTemplate {
   BasicDataCentralImportCandidateMerge = '/basic-data/central-import/:candidateIdentifier/merge/:registrationIdentifier',
   BasicDataInstitutions = '/basic-data/institutions',
   BasicDataNvi = '/basic-data/nvi',
+  BasicDataNviNew = '/basic-data/nvi/new',
```

Comment:
Kan argumentere for at `/nvi-periods/` hadde vært bedre pathname her, men syns samtidig at det gir litt mer støy i URL. I tvil, men lener mot bare `nvi`